example.yaml

```diff
@@ -1,81 +1,163 @@
 ---
-api_hostname: https://app.datadoghq.com
 metric_max_length: 4096
 trace_max_length_bytes: 16384
 flush_max_per_body: 25000
-debug: true
 enable_profiling: false
 interval: "10s"
-key: "farts"
+
+# == PERFORMANCE ==
+
+# The size of the buffer we'll use to buffer socket reads. Tune this if you
+# you think Veneur needs more room to keep up with all packets.
+read_buffer_size_bytes: 2097152
+
+# Adjusts the number of workers Veneur will distribute aggregation across.
+# More decreases contention but has diminishng returns.
+num_workers: 96
+
 # Numbers larger than 1 will enable the use of SO_REUSEPORT, make sure
 # this is supported on your platform!
-num_workers: 96
 num_readers: 1
+
+# The number of SSF packets that can be processed
+# per flush interval
+ssf_buffer_size: 16384
```

Comment:
Don't we also need to change https://github.com/stripe/veneur/blob/b5d18d133f202fe605d45ecdec8c91c45af6dbda/worker.go#L16?